docs/hardware/devices/seeed-wm1110/index.mdx

```diff
@@ -67,11 +68,7 @@ Basic support for this device has only just been added, and not all features wor
 
 <TabItem value="wio-tracker-wm1110">
 
-## Seeed Wio Tracker 1110
-
-:::caution Firmware Version Notice
-Basic support for this device has only just been added, and not all features work
-:::
+## Wio Tracker 1110 Dev Kit for Meshtastic
```

Comment:
Being picky here, I know, but this doesn't need `for Meshtastic` on the end. We're already on the meshtastic website and since this shows up in the sidebar it's better to keep it shorter. It will also match the title from the other section

Reply:
@tekstrand , there is a similarly named but different product, the "Wio Tracker 1110 Dev Board". I used the full name of the product here to distinguish the two.

Reply:
Feel free to fix with that in mind, otherwise I'll get to it next week.